tests/page-objects/reports/reports.wdio.page.js

```diff
@@ -212,6 +217,18 @@ const getCurrentReportId = async () => {
   return currentUrl.slice(reportBaseUrl.length);
 };
 
+const getReportDetailFieldValueByLabel = async (label) => {
+  await reportBodyDetails().waitForDisplayed();
+  const fieldsCount = (await reportDetailsFields()).length;
+  for (let i = 1; i <= fieldsCount; i++) {
```

Comment:
Can you instead use: 
```javascript
for (const field of await await reportDetailsFields()) 
```
and not use indexes at all? 

You could also get inspiration from the `contacts.wdio.page.js` file, where we have a function that gets a contact summary field by name: `getContactSummaryField`.

Reply:
Updated, thanks for the suggestion.

Reply:
Thanks for the tip. I've made the change.